README.md

```diff
@@ -1,3 +1,7 @@
+Hi 
```

Comment:
Hi Azamat!

Though you managed to raise a PR, it modifies the wrong file. Please remove these changes and add your name to the file [`students/students1.list`](https://github.com/wedoforce/students/blob/main/students/students1.list)